tester.py

```diff
@@ -0,0 +1,169 @@
+from pyband.wallet import Ledger
```

Comment:
Refactor this file to folder examples, split each main to separate file and named it (move old `example.py` too)